src/lib/components/ResultsGrid/ResultsGrid.js

```diff
@@ -71,7 +71,7 @@ const Element = ({ overridableId, results, resultsPerRow }) => {
   return (
     <Overridable
       id={buildUID("ResultsGrid.container", overridableId)}
-      results={results}
+      results={_results}
```

Comment:
@kpsherva @jrcastro2 given that you have reviewed this one.
IMO, I don't think this was 100% correct: the idea here is to pass the "raw" values, that would allow you to use `results` to build your own layout when overriding.
Passing `_results` means that when overriding, you get already a `GridItem` React cmp, instead of the list of `results` object, and you cannot easily change that cmp.

Can you please fix this?

Ping @zzacharo for any other opinion.